lib/vagrant-service-manager/command.rb

```diff
@@ -6,8 +6,7 @@ module Vagrant
   module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
-    SCCLI_SERVICES = ['openshift', 'k8s']
-    KUBE_NAMES = ['kubernetes', 'k8s']
+    SCCLI_SERVICES = SUPPORTED_SERVICES.drop(1)
```

Comment:
Can we set the `SCCLI_SERVICES` explicitly instead of operating on `SUPPORTED_SERVICES`
like
```
SUPPORTED_SERVICES = ["openshift", "kubernetes"]
```